core/src/main/java/feign/RequestTemplate.java

```diff
@@ -728,6 +728,36 @@ public RequestTemplate header(String name, Iterable<String> values) {
     return appendHeader(name, values);
   }
 
+  /**
+   * @see RequestTemplate#headerLiteral(String, Iterable)
+   */
+  public RequestTemplate headerLiteral(String name, String... values) {
+    if (values == null) {
+      return appendHeader(name, Collections.emptyList(), true);
```

Comment:
Any reason not to call `headerLiteral` here?

Reply:
No specific reason other then to align this code with the implementation of RequestTemplate#header(...)